applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -79,7 +79,8 @@ def GetDefaultParameters(cls):
                 "max_substeps" : 0,
                 "levelset_splitting" : false,
                 "eulerian_error_compensation" : false,
-                "cross_wind_stabilization_factor" : 0.7
+                "cross_wind_stabilization_factor" : 0.7,
+                "algebraic_stabilization" : false
```

Comment:
This is becoming a bit messy as the settings of different levelset convection methods are mixed up. This could be even worse in the @KratosMultiphysics/altair side since as @ddiezrod said they have more elements.

At this point, I suggest to use an "element_type" field instead of the `cross_wind_stabilization_factor` and the `algebraic_stabilization` ones in the solver. Then, this field can be used by the levelset convection process to get the prototypes.

I think this will ease the usage.

(On top of this, we still need to think about the `use_bfecc` and related fields. Most probably we can remove them from the solver as the performance of the eulerian one is reported to be much better in this case. This is of course to be discussed in a future PR).